<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>QHostInfo</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_getLocalHost">
           <property name="text">
            <string>获取本机主机名和地址</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>域名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_name">
             <item>
              <property name="text">
               <string>www.baidu.com</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>www.ptpress.com.cn</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>www.facebook.com</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>www.google.com</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>www.douban.com</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_findNameIP">
           <property name="text">
            <string>查找域名的IP地址</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>QNetworkInterface</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_allAddresses">
           <property name="text">
            <string>allAddresses()</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_allInterfaces">
           <property name="text">
            <string>allInterfaces()</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>只显示IPv4地址</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
